package.json

```diff
@@ -53,6 +53,13 @@
     "wait-on": "7.2.0"
   },
   "pnpm": {
+    "packageExtensions": {
+      "@typescript-eslint/eslint-plugin": {
+        "dependencies": {
+          "typescript": "5.0.3"
```

Comment:
Temporary workaround until galex is updated, to prevent an unwanted change in the lockfile and ensure that `@typescript-eslint` runs with a version of typescript it supports.